src/utils/optionsparser.cc

```diff
@@ -120,6 +120,9 @@ bool OptionsParser::ProcessAllFlags() {
 
 bool OptionsParser::ProcessFlags(const std::vector<std::string>& args) {
   auto show_help = false;
+  if (CommandLine::BinaryName().find("expert") != std::string::npos) {
```

Comment:
Feel free also to pick some funnier/mysterious substring for that. :)

Reply:
"galaxybrain"

Reply:
I was leaning towards "pro" for a more ironic approach.

Reply:
Have no objections to either of options. Or `hackerman` to get "pro" irony even further. :)

Reply:
I propose `sudo`, as changing hidden options should require knowledge about the inner workings and be done on your own risk.